tests/test_importer_utils.py

```diff
@@ -303,6 +303,16 @@ def test_dict_to_iso_date_year_month(self):
 
 class TestWikitext(unittest.TestCase):
 
+    def test_get_urls_none(self):
+        text = "[[Tegera Arena]], huvudentrén"
+        output = []
+        self.assertEqual(utils.get_external_links(text), output)
+
+    def test_get_urls_one(self):
```

Comment:
add one more small test with something like "prefix [url] postfix" i.e. a url which would look like [1] on a wiki. And amend this one to also have a pre- and postfix.

Reply:
Done